spec/formatting.md

```diff
@@ -721,6 +725,26 @@ These are divided into the following categories:
     > when * {The value is not one.}
     > ```
 
+  - **Unsupported Expression errors** occur when an expression uses
```

Comment:
The current plural language matches that used by all the other errors. Could fixes to the style used for the error descriptions be handled as a separate PR, rather than further increasing the scope of this PR?

Reply:
I tend to favor "not digging the hole deeper" when committing text, but since we'll probably merge this in an hour, I can come back and make a more holistic set of edits later.

With regard to this comment (and noting that it rhymes with other comments in other PRs), the style I prefer is something like:

> **Term** is (definition here).
>
> (discussion of _term_'s attributes, usage, triggers, etc., with _term_ referencing the definition)

I tend to prefer singular, because automatic linking of references to the definition is easier when the term is originally singular. Unlike W3C, we don't have markup to use for alternate forms of the term and will have to write the link generation eventually as a script (if we decide we want linking). 🤷